doc/modules/ROOT/pages/tutorials/heterogeneous-node-classification-with-hashgnn.adoc

```diff
@@ -0,0 +1,377 @@
+// DO NOT EDIT - AsciiDoc file generated automatically
```

Comment:
Do we need to check in these files? Is generating them time-consuming/expensive? If not, perhaps we should look into generating these "on the fly" instead of checking them in?

Advantage of checking in: we control change explicitly
Advantage of generating on the fly: changes made to the generator are effective immediately

Yeah I prefer checking them in for now. Even if we risk editing these files by mistake (I bet this will happen)